pyqtgraph/graphicsItems/ViewBox/ViewBox.py

```diff
@@ -1573,6 +1572,14 @@ def updateViewRange(self, forceX=False, forceY=False):
                     maxRng[axis] = min(maxRng[axis], limits[axis][1] - limits[axis][0])
                 else:
                     maxRng[axis] = limits[axis][1] - limits[axis][0]
+        for axis in [0,1]:
```

Comment:
Can this line be removed and incorporate the logic added into the for loop above?